src/__global.zig

```diff
@@ -93,7 +93,24 @@ pub fn exit(code: u8) noreturn {
     runExitCallbacks();
     Output.flush();
     std.mem.doNotOptimizeAway(&Bun__atexit);
-    std.c.exit(@intCast(code));
+    std.c.exit(code);
+}
+
+pub fn raiseIgnoringPanicHandler(sig: anytype) noreturn {
+    Output.flush();
+    @import("./crash_reporter.zig").on_error = null;
+    if (sig >= 1 and sig != std.os.SIG.STOP and sig != std.os.SIG.KILL) {
+        const act = std.os.Sigaction{
+            .handler = .{ .sigaction = @ptrCast(@alignCast(std.os.SIG.DFL)) },
+            .mask = 0,
+            .flags = 0,
+        };
+        std.os.sigaction(@intCast(sig), &act, null) catch {};
+    }
+    // TODO(@paperdave): report a bug that this intcast shouldnt be needed. signals are i32 not u32
+    // after that is fixed we can make this function take i32
+    _ = std.c.raise(@intCast(sig));
```

Comment:
Why are we raising and then aborting?

aborting causes an exit code of 1.

Reply:
this is to do *something* in the case the process is not dead after `raise` is called. since `raise` in zig is not `noreturn` and it returns something, this implies there is some possibility for it to not end the process.

> aborting causes an exit code of 1

this is not true.

<img width="609" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/949b2f54-a338-4428-83f8-b63417945cd1">

please see the [manpage on abort(3)](https://man7.org/linux/man-pages/man3/abort.3.html)